fplll/test.cpp

```diff
@@ -19,17 +19,47 @@
 using namespace std;
 using namespace fplll;
 
+
```

Comment:
Is editing test.cpp a mistake? (I'd guess this file should be removed, in general)